src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+
+    public Figure getRandomFigure() {
+        Random random = new Random();
+        int figureType = random.nextInt(5);
+        switch (figureType) {
+            case 0:
+                return new Square(random.nextInt(10) + 1, ColorSupplier.getRandomColor());

```

Comment:
create a utility method for creating of geometric property for this logic - `random.nextInt(10) + 1`